featurebyte/routes/registry.py

```diff
@@ -102,131 +103,145 @@
+app_container_config.register_class(TableColumnsInfoService)
 app_container_config.register_class(
     TableController, dependency_override={"service": "table_service"}
 )
 app_container_config.register_class(TableInfoService)
+app_container_config.register_class(TableService)
+app_container_config.register_class(TableStatusService)
 app_container_config.register_class(
     TargetController, dependency_override={"service": "target_service"}
 )
+app_container_config.register_class(TargetService)
 app_container_config.register_class(
     TargetNamespaceController, dependency_override={"service": "target_namespace_service"}
 )
+app_container_config.register_class(TargetNamespaceService)
+app_container_config.register_class(TaskController)
+app_container_config.register_class(TempDataController)
+app_container_config.register_class(TileCacheService)
 app_container_config.register_class(TileColumnTypeExtractor)
+app_container_config.register_class(TileManagerService)
+app_container_config.register_class(TileSchedulerService)
 app_container_config.register_class(TileTaskExecutor)
 app_container_config.register_class(
     UserDefinedFunctionController, dependency_override={"service": "user_defined_function_service"}
 )
+app_container_config.register_class(UserDefinedFunctionService)
+app_container_config.register_class(UserService)
+app_container_config.register_class(VersionService)
+app_container_config.register_class(ViewConstructionService)
+app_container_config.register_class(WorkingSchemaService)
 
 
 # These have force_no_deps set as True, as they are manually initialized.
-app_container_config.register_class(MongoBackedCredentialProvider, force_no_deps=True)
-app_container_config.register_class(TaskController, force_no_deps=True)
-app_container_config.register_class(TaskManager, force_no_deps=True)
-app_container_config.register_class(TempDataController, force_no_deps=True)
 app_container_config.register_class(Persistent, force_no_deps=True)
+app_container_config.register_class(Storage, force_no_deps=True)
+app_container_config.register_class(Storage, force_no_deps=True, name_override="temp_storage")
+app_container_config.register_class(TaskManager, force_no_deps=True)
+app_container_config.register_class(User, force_no_deps=True)
+
+
+class CatalogId:
+    """
+    This is a special placeholder class that is used to inject the container config.
+    """
+
+
+# This looks a little funny right now, but every entry in the instance map must currently be found in the
+# app_container_config, as some validation checks depend on it. For this particular case, we inject the
+# catalog_id directly into the `instance_map` in the LazyAppContainer constructor. As such, we need an item in the
+# app_container_config that is called `catalog_id`. This class of CatalogId will get parsed into `catalog_id`, and
+# as such, works as a placeholder.
+app_container_config.register_class(CatalogId, force_no_deps=True)
```

Comment:
i'll try to clean this up in a follow-up, but works ok for now, and isn't too common.